warehouse/warehouse.go

```diff
@@ -790,9 +789,9 @@ func (wh *HandleT) getUploadsToProcess(availableWorkers int, skipIdentifiers []s
 						%s t
 					WHERE
 						t.destination_type = '%s' and t.in_progress=%t and t.status != '%s' and t.status != '%s' %s and COALESCE(metadata->>'nextRetryTime', now()::text)::timestamptz <= now()
-				) grouped_uplaods
+				) grouped_uploads
```

Comment:
Has the column in the database also been renamed? How do we handle deployments that require schema changes?